packages/react-frontend/src/routes/FooterFiles/faqs.jsx

```diff
@@ -10,6 +10,7 @@ export default function Faqs() {
 
```

Comment:
I pulled your branch and looked at your FAQ page. Your footer cuts off the bottom part of your FAQ page.

Reply:
Yes, I noticed that! I just fixed it. Thanks for your input!